src/components/TableResize.js

```diff
@@ -98,7 +95,18 @@ class TableResize extends React.Component {
     const { resizeCoords, tableWidth } = this.state;
 
     Object.entries(resizeCoords).forEach(([key, item]) => {
-      let newWidth = Number(((item.left - lastPosition) / tableWidth) * 100).toFixed(2);
+      let newWidth = Number(((item.left - lastPosition) / tableWidth) * 100);
```

Comment:
Decimal points are likely to cause bugs.

Can't we just throw away the decimal point?
I think we can throw away the decimal point.

```js
const newWidth = Number(((item.left - lastPosition) / tableWidth) * 100).toFixed(0)
```

Reply:
I've done a little more research on this and decimal places will [not cause issues](https://cruft.io/posts/percentage-calculations-in-ie/). I was mostly just being overly cautious. What perplexed me is why the code was rounding to 2 decimal places. It appears that in the old days browsers rounded to 2 decimal places, so possibility for consistency it was set to round to 2. 

For me (Chrome/Firefox/Safari) on Mac, there is a noticeable jitteriness when moving the dividers when only 2 decimal places are used. When there is no cap (and the browser does the rounding), it looks nice and smooth. Anyway, since a longer decimal place won't cause an issue, I think I'm going to leave it be. Certain browsers will round the number differently, but a larger decimal point allows newer browsers to be more precise.